### ISSUE TITLE
Iceberg & NAG libraries: remove env var?

### ISSUE THREAD

**Issue by willfurnass** [rcgsheffield/sheffield_hpc#428]
Test @mikecroucher 's solution for removing the need for specifying `-I$NAGINC ` when using the PGI version of the NAG libraries.  Then update [the relevant page of the docs](https://github.com/rcgsheffield/sheffield_hpc/blob/7ce497c761275507eddf1ae630d14bb101a5d33c/iceberg/software/libs/nagfortran.rst).

Previously discussed on PR #423

**Comment by cs1ds** [comment]
Mike, I have tested the pgi 15.10 compiler and using the NAG library. It works without specifying -I$NAGINC.   I noticed that this is not yet the release version of the compiler. 

 We have two options at this point:
  (1) Implement your correction for the 15.07 and the earlier versions of the compilers 
  (2) Make 15.10 the release version and make sure that NAG Library access  info specifies using 15.10 and newer. 
  Although we now have the NAG library accessible from 15.07 my choice would be to go for OPTION(2).  We can then write a consistent document without the mention of the I$NAGINC .
Are there any impediments to making 15.10 the default version ? ( MPI perhaps ? )
